media-plugins/distrho-ports/distrho-ports-99999999.ebuild

```diff
@@ -22,41 +22,26 @@ REQUIRED_USE="|| ( lv2 vst )"
 
 RDEPEND="media-libs/alsa-lib
 	media-libs/freetype
-	media-libs/ladspa-sdk
 	virtual/opengl
 	x11-libs/libX11
 	x11-libs/libXext
 	x11-libs/libXcursor
 	x11-libs/libXrender"
-DEPEND="${RDEPEND}
-	dev-util/premake:3"
-
-QA_PRESTRIPPED="
-	/usr/lib/lv2/.*
-	/usr/lib/vst/.*
-"
+DEPEND="${RDEPEND}"
 
 src_prepare() {
-	default
-	scripts/premake-update.sh linux
-}
+	# Remove stripping of binaries
+	sed -i -e "/'-Wl,--strip-all',/d" meson.build || die "sed failed"
 
-src_compile() {
-	if use lv2; then
-		emake lv2
-	fi
-	if use vst; then
-		emake vst
-	fi
+	default
 }
 
-src_install() {
-	emake DESTDIR="${D}" PREFIX="/usr" install
-	if use !lv2; then
-		rm -rf "${D}"/usr/lib/lv2
-	fi
-	if use !vst; then
-		rm -rf "${D}"/usr/lib/vst
-	fi
-	rm -rf "${D}"/usr/src
+src_configure() {
+	local emesonargs=(
+		-Doptimizations=false
+		$(meson_use vst build-vst2)
```

Comment:
I figured there wouldn't be much to gain from splitting this so I used a single `vst` USE flag for both vst2 and vst3

Reply:
No, it's fully provided by the JUCE libs that are included